src/dkg_polynomial.erl

```diff
@@ -20,6 +31,23 @@ generate(Pairing, T, Index, Term) ->
     NewHead = erlang_pbc:element_add(erlang_pbc:element_sub(Head, Val), Term),
     [NewHead | Tail].
 
+degree(Poly) ->
```

Comment:
Still uncomfortable with this change. Shouldn't we stick to the mathematical definition?